sdks/java/harness/src/main/java/org/apache/beam/fn/harness/control/ProcessBundleHandler.java

```diff
@@ -341,53 +341,52 @@ private void createRunnerAndConsumersForPTransformRecursively(
       throws Exception {
     BundleProcessor bundleProcessor =
         bundleProcessorCache.find(request.getProcessBundleProgress().getInstructionId());
-    if (bundleProcessor == null) {
```

Comment:
nit, you could remove the level of nesting by using a guard statement (here and below):
```
if (bundleProcessor == null) {
  return BeamFnApi.InstructionResponse.newBuilder().setProcessBundleProgress(BeamFnApi.ProcessBundleProgressResponse.getDefaultInstance());
}
```

Reply:
Done.